### PR TITLE
Immediately clear Inovelli notificationComplete events

### PR DESCRIPTION
**Pull request by wingrunr21**
This is an alternative proposal to address the issues raised in Koenkk/zigbee2mqtt#31686 (via wbyoung/lampie#2)

This effectively vendors the behavior to immediately clear the `notificationComplete` event here. Rationale is this is vendor-specific behavior and probably belongs here vs in Z2M via the HA extension.

Chose `0` on the timeout just so it fires next tick.

Made with my buddy Claude

cc @rohankapoorcom @wbyoung

### CODE REVIEW

**Comment by wbyoung** [comment]
> This is an alternative proposal to address the issues raised in [Koenkk/zigbee2mqtt#31686](https://github.com/Koenkk/zigbee2mqtt/pull/31686) (via [wbyoung/lampie#2](https://github.com/wbyoung/lampie/pull/2))
> 
> This effectively vendors the behavior to immediately clear the `notificationComplete` event here. Rationale is this is vendor-specific behavior and probably belongs here vs in Z2M via the HA extension.
> 
> Chose `0` on the timeout just so it fires next tick.
> 
> Made with my buddy Claude
> 
> cc @rohankapoorcom @wbyoung

Not being a Z2M user, take my feedback lightly, but as an Inovelli switch owner, this is how I'd expect it to work.

**Comment by Koenkk** [comment]
@rohankapoorcom  @InovelliUSA  could you take a look?

**Comment by rohankapoorcom** [comment]
Approach seems reasonable to me. @Koenkk are you comfortable with introducing a timer element though in ZHC?

File: test/inovelli.test.ts [hunk 1]
```diff
@@ -1700,9 +1711,34 @@ describe("Inovelli fromZigbee converters", () => {
...
+                    definition,
+                    // biome-ignore lint/suspicious/noExplicitAny: test mock
+                    {data: {notificationType: 16}, endpoint: device.getEndpoint(1)} as any,
+                    (p) => published.push(p),
+                    {},
+                    {state: {}, device: null, deviceExposesChanged: () => {}},
+                );
+                await new Promise((resolve) => setTimeout(resolve, 0));
+                expect(published).toEqual([{notificationComplete: ""}]);
+            });
```
[the remark this answers is not in the document]
Comment (rohankapoorcom) [reply test/inovelli.test.ts:1741]: This is probably needed to avoid flaky unit tests.

**Comment by Koenkk** [comment]
> Approach seems reasonable to me. @Koenkk are you comfortable with introducing a timer element though in ZHC?

For edge cases it's fine, it's not the first one.

**Comment by Nerivec** [comment]
@Koenkk sidenote, we might want to `unref()` all timers created in ZHC.
https://nodejs.org/api/timers.html#timeoutunref
